yarn.lock

```diff
@@ -13608,6 +13574,62 @@ react-native-webview@5.4.6:
     escape-string-regexp "1.0.5"
     invariant "2.2.4"
 
+react-native@0.59.5:
```

Comment:
This doesn't seem right - you maybe need to check out the submodules before running Yarn.

Reply:
That's what I thought too, I'll clean my system and retry it tonight. Thanks for the tip! ❤️

Reply:
Turned out some unexplainable yarn config from another project to make it run. Cleaned it, removed it and now it looks way better!